app/src/util/api.ts

```diff
@@ -147,6 +149,39 @@ const organizationContextCheck = async ({
   }
 };
 
+function getBearerToken(header: string): any {
+  const match = header.match(/^Bearer\s+(.*)$/)
+  if (!match) {
+    throw new createHttpError.BadRequest(`Malformed Authorization header`)
+  }
+  if (!process.env.VERIFICATION_TOKEN_SECRET) {
+    logger.error("Need to assign VERIFICATION_TOKEN_SECRET in env!");
+    throw createHttpError.InternalServerError("Configuration error");
+  }
+  return jwt.verify(
+    match[1],
+    process.env.VERIFICATION_TOKEN_SECRET
+  )
```

Comment:
Do we need to handle errors here by throwing e.g. a 401 error when they occur and provide a clear response?

Reply:
Yes